src/clj/puppetlabs/services/jruby/jruby_metrics_core.clj

```diff
@@ -2,6 +2,7 @@
   (:require [schema.core :as schema]
             [puppetlabs.metrics :as metrics]
             [puppetlabs.services.jruby-pool-manager.jruby-schemas :as jruby-schemas]
+            [clojure.string :as string]
```

Comment:
nit, almost everyone else we use `:as str` even though that shadows `clojure.core/str`. I *think* our require is idiomatic but going to check in the clojurians channel.

Reply:
Looks like Puppet Server is split with 7 instances of `:as str` and 4 instances of `:as string`. Happy to switch to `:as str` as that seems to be the preference.

Reply:
Replaced with `:as str`.